modules/meeting/app/controllers/meetings_controller.rb

```diff
@@ -123,14 +128,24 @@ def set_time_zone(&)
 
   def build_meeting
     @meeting = Meeting.new
+    if meeting_params.present?
+      convert_params
+      @meeting.participants.clear # Start with a clean set of participants
+      @meeting.participants_attributes = @converted_params.delete(:participants_attributes)
+      @meeting.attributes = @converted_params
+    end
     @meeting.project = @project
     @meeting.author = User.current
```

Comment:
Not necessary for this PR but it would make sense to move this functionality into a `Meetings::SetAttributesService`.